src/util-ioctl.c

```diff
@@ -114,11 +114,15 @@ int GetIfaceMTU(const char *dev)
  * for the link. In case of uncertainty, it will output a
  * majorant to be sure avoid the cost of dynamic allocation.
  *
- * \param Name of a network interface
+ * \param LiveDevice object
  * \retval 0 in case of error
  */
-int GetIfaceMaxPacketSize(const char *dev)
+int GetIfaceMaxPacketSize(LiveDevice *ld)
```

Comment:
Why do we need this change ? (to put in the commit message I guess)

Reply:
ok. Putting in commit message